c_binding/include/libstoragemgmt/libstoragemgmt_local_disk.h

```diff
@@ -73,6 +73,37 @@ int LSM_DLL_EXPORT lsm_local_disk_vpd83_get(const char *sd_path,
                                             char **vpd83,
                                             lsm_error **lsm_err);
 
+/**
+ * Query the disk rotation speed - revolutions per minute(RPM) of given SCSI
+ * disk path.
+ * Require permission to open disk path(root user or disk group).
+ * New in version 1.3.
+ * @param[in]  disk_path
+ *                      String. The path of SCSI disk block, example "/dev/sdb".
+ * @param[out] rpm      Output pointer of int32_t.
+ *                          -1 (LSM_DISK_RPM_UNKNOWN):
+ *                              Unknown RPM
+ *                           0 (LSM_DISK_RPM_NON_ROTATING_MEDIUM):
+ *                              Non-rotating medium (e.g., SSD)
+ *                           1 (LSM_DISK_RPM_ROTATING_UNKNOWN_SPEED):
+ *                              Rotational disk with unknown speed
+ *                          >1:
+ *                              Normal rotational disk (e.g., HDD)
+ * @param[out] lsm_err
+ *                      Output pointer of lsm_error. Error message could be
+ *                      retrieved via lsm_error_message_get(). Memory should be
+ *                      freed by lsm_error_free().
+ * @return LSM_ERR_OK                   On success.
+ *         LSM_ERR_INVALID_ARGUMENT     When any argument is NULL.
+ *         LSM_ERR_LIB_BUG              When something unexpected happens.
+ *         LSM_ERR_NOT_FOUND_DISK       When provided disk path not found.
+ *         LSM_ERR_PERMISSION_DENIED    No sufficient permission to access
+ *                                      provided disk path.
```

Comment:
Since we are `lsm_local_disk_xxx()`, I believe we could assume NVMe is also a disk. We could change to `lsm_local_block_xxx()` if that matters.